docs/roadmap.md

```diff
@@ -3,9 +3,9 @@
 ## Getting to 0.0.1
 
 - First release contains:
-  - All service/client underpinnings
-  - Extension support
-  - JobStateUpdateHandler can be replaced by DiracX
+    - All service/client underpinnings
+    - Extension support
+    - JobStateUpdateHandler can be replaced by DiracX
```

Comment:
This roadmap is internal to diracx, it does not need to be in an official documentation